org.eclipse.jdt.ls.core/pom.xml

```diff
@@ -30,6 +30,47 @@
 					</artifactItems>
 				</configuration>
 			</plugin>
+			<plugin>
+				<groupId>org.codehaus.gmaven</groupId>
+				<artifactId>groovy-maven-plugin</artifactId>
+				<version>2.1.1</version>
+				<executions>
+					<execution>
+						<phase>generate-resources</phase>
+						<goals>
+							<goal>execute</goal>
+						</goals>
+						<configuration>
+							<source>
+								import groovy.json.JsonSlurper;
+								import groovy.json.JsonOutput;
+								URL versionUrl = new URL('https://services.gradle.org/versions/all');
+								String versionStr = versionUrl.text;
+								File outFile = new File("${basedir}/gradle/checksums/versions.json");
+								outFile.write(versionStr);
+								List versions = new JsonSlurper().parseText(versionStr);
+								class Checksum { String wrapperChecksumUrl; String sha256 };
+								List checksums = [];
+								for (version in versions) {
+									if (version.wrapperChecksumUrl != null) {
+										def sha256url = new URL(version.wrapperChecksumUrl);
+										try {
+											def sha256 = sha256url.text;
+											def checksum = new Checksum(wrapperChecksumUrl: version.wrapperChecksumUrl, sha256: sha256);
+											checksums.add(checksum);
+										} catch(Exception e) {
+											// ignore 
+										}
+									}
+								}
+								outFile = new File("${basedir}/gradle/checksums/checksums.json");
+								def json = JsonOutput.toJson(checksums);
+								outFile.write(JsonOutput.prettyPrint(json));
+							</source>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
```

Comment:
```xml
<plugin>
				<groupId>org.codehaus.gmaven</groupId>
				<artifactId>groovy-maven-plugin</artifactId>
				<version>2.1.1</version>
				<executions>
					<execution>
						<?m2e execute onConfiguration?>
						<phase>generate-resources</phase>
						<goals>
							<goal>execute</goal>
						</goals>
						<configuration>
							<source>
								import groovy.json.JsonSlurper
								import groovy.json.JsonOutput
								
								def versionUrl = new URL("https://services.gradle.org/versions/all")
								def versionStr = versionUrl.text;
								def versionsFile = new File("${basedir}/gradle/checksums/versions.json")
								versionsFile.parentFile.mkdirs() //just in case
								versionsFile.write(versionStr)
								
								def versions = new JsonSlurper().parseText(versionStr);
								class Checksum { String wrapperChecksumUrl; String sha256 }
								
								def checksums = []
								versions.each {
									if (it.wrapperChecksumUrl) {
										def sha256url = new URL(it.wrapperChecksumUrl)
										try {
											def sha256 = sha256url.text
											def checksum = new Checksum(wrapperChecksumUrl: it.wrapperChecksumUrl, sha256: sha256);
											checksums.add(checksum)
										} catch(Exception e) {
											// ignore 
										}
									}
								}
								def json = JsonOutput.toJson(checksums)
								def checksumsFile = new File("${basedir}/gradle/checksums/checksums.json")
								checksumsFile.write(JsonOutput.prettyPrint(json))
							</source>
						</configuration>
					</execution>
				</executions>
			</plugin>
```

Reply:
Done.